packages/shared/src/common/metriport-organization.ts

```diff
@@ -0,0 +1,21 @@
+import { Organization } from "@medplum/fhirtypes";
+export const MetriportOrganization: Organization = {
```

Comment:
Constants should be camelCase

nit: add empty line between import and function